CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ### Features
 
+* Added alpha diversity metric `sobs`, which is the observed species richness (S_{obs}) of a sample. It means to replace `observed_otus` which uses the historical term "OTU". Also added metric `observed_features` to be compatible with the QIIME 2 terminology. All three metrics are equivalent ([#1902](https://github.com/biocore/scikit-bio/pull/1902)).
```

Comment:
@qiyunzhu The sentence `It means to replace 'observed_otus' ...` could be clearer if we said `'sobs' will replace 'observed_otus', ...`, or `'sobs' replaces 'observed_otus', ...`. Also, note the comma before the word `which`.

Reply:
Thanks @mataton I guess you can suggest changes in the GitHub web interface. Would like to give it a try?